addons/delivery_mondialrelay/data/data.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="product_product_delivery_mondialrelay" model="product.product">
+            <field name="name">Mondial Relay</field>
+            <field name="default_code">MR</field>
+            <field name="type">service</field>
+            <field name="categ_id" ref="delivery.product_category_deliveries"/>
+            <field name="sale_ok" eval="False"/>
+            <field name="purchase_ok" eval="False"/>
+            <field name="list_price">5</field>
+            <field name="invoice_policy">order</field>
+        </record>
+
+        <record id="delivery_carrier_mondialrelay_be_lu" model="delivery.carrier">
+            <field name="name">Mondial Relay</field>
+            <field name="sequence">30</field>
+            <field name="delivery_type">base_on_rule</field>
+            <field name="integration_level">rate</field>
+            <field name="country_ids" eval="[(6, 0, [ref('base.be'), ref('base.lu')])]" />
+            <field name="product_id" ref="delivery_mondialrelay.product_product_delivery_mondialrelay"/>
+        </record>
+        <record id="delivery_price_rule1" model="delivery.price.rule">
```

Comment:
maybe put data of `delivery.price.rule` in demo file, if not after each update module user modification are canceled.

Reply:
We don't move real data in demo file to avoid modification during update :grimacing: 
But yes, mayve a noupdate on data to consider

Reply:
but with no update you cannot remove a line, it can be an issue later.